aspnetcore/signalr/dotnet-client/sample/SignalRChatClient/MainWindow.xaml.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Microsoft.AspNetCore.SignalR.Client;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Windows;
+
+namespace SignalRChatClient
+{
+    public partial class MainWindow : Window
+    {
+        static HubConnection connection;
+        public MainWindow()
+        {
+            InitializeComponent();
+
+            connection = new HubConnectionBuilder()
+            .WithUrl("https://localhost:44317/ChatHub")
+            .ConfigureLogging(logging => logging.AddConsole())      
```

Comment:
This requires another package reference. We should remove it.

Reply:
Logging is a section of the doc, and it's easy enough to direct users to add the `Microsoft.Extensions.Logging.Console` package.

Reply:
It shows that at the bottom, it should be removed in the windows forms app (because there's no console)

Reply:
Gotcha